drivers/node/node_registry.go

```diff
@@ -77,6 +77,16 @@ func GetMasterNodes() []Node {
 	return nodeList
 }
 
+// IsMasterNode returns true is node is Masternode
```

Comment:
Please correct the comment, it should be `IsMasterNode returns true if node is a Masternode` instead of `IsMasterNode returns true is node is Masternode`

Reply:
fixed